packages/front-end/pages/environments.tsx

```diff
@@ -222,12 +222,10 @@ const EnvironmentsPage: FC = () => {
                                 const newEnvs = [...environments];
                                 newEnvs.splice(i, 1);
                                 newEnvs.splice(i - 1, 0, e);
-                                await apiCall(`/organization`, {
+                                await apiCall(`/environment/order`, {
```

Comment:
Do we need a net new endpoint for this? Should we simply allow the plural endpoint to also update the order?

Reply:
I ultimately broke it out into a separate endpoint as the permission logic for each of these is different.

For re-ordering, the user only needs to have permission to create environments globally, or in at least 1 project. Whereas, if the user is updating the actual environments, they need create/update permission for every endpoint passed in.

If we added this to the plural endpoint, the logic would get really complicated. We'd need to do a deep comparison of the environments being passed in to see if the user was just trying to update the order, or update the actual environments. 

Ultimately, it felt like it would lead to bugs, and was easily handled by breaking the logic out into two separate endpoints.